src/AutoRest.CSharp/Common/AutoRest/Plugins/NewProjectScaffolding.cs

```diff
@@ -110,7 +110,6 @@ private string GetAssemblyInfo()
 [assembly: InternalsVisibleTo(""{0}.Tests{1}"")]
 {2}";
             const string azureResourceProvider = @"
-
```

Comment:
I think we should leave this, as the file needs to be hand-edited after being generated and the whitespace helps with readability.

Reply:
same as above.